vaadin-spring/src/main/java/com/vaadin/flow/spring/SpringBootAutoConfiguration.java

```diff
@@ -100,4 +98,13 @@ public ServerEndpointExporter websocketEndpointDeployer() {
         return new VaadinWebsocketEndpointExporter();
     }
 
+    /**
+     * Makes the request util available.
```

Comment:
Why ?
For what?
Where this class is used ?

Reply:
This code is not from me. It came when I merged the master branch

Reply:
The code may not be here if you haven't changed the code: the PR shows a difference between the `master` branch and your branch.

So it's either something wrong with the PR/branch or the branch is out of sync. You should rebase it on top of the master.